README.md

```diff
@@ -80,86 +80,17 @@ size = "small" | "medium" | "large" | [( "2" | "4" | "8" )] , "xlarge";
 
 # Series
 
-.                           |  C    |  CO   |  CX   |  GN   |  M    |  N
+.                           |  CX   |  GN   |  M    |  N    |  U    |  UO
 ----------------------------|-------|-------|-------|-------|-------|------
-*Has GPUs*                  |       |       |       |  ✓    |       |
-*Hugepages*                 |       |       |       |       |  ✓    |  ✓
-*Cache backed RAM*          |       |  ✓    |       |       |       |
-*Dedicated CPU performance* |       |       |  ✓    |       |       |
-*Burstable CPU performance* |  ✓    |  ✓    |       |  ✓    |  ✓    |  ✓
-*Isolated emulator threads* |       |       |  ✓    |       |       |
-*vNUMA*                     |       |       |  ✓    |       |       |
+*Has GPUs*                  |       |  ✓    |       |       |       |
+*Hugepages*                 |       |       |  ✓    |  ✓    |       |
+*Cache backed RAM*          |       |       |       |       |       |  ✓
+*Dedicated CPU performance* |  ✓    |       |       |       |       |
+*Burstable CPU performance* |       |  ✓    |  ✓    |  ✓    |  ✓    |  ✓
```

Comment:
Is `Burstable CPU performance` a feature the user or admin wants to know about? Doesn't it mean just "shared CPU", which means no dedicated CPU? Can this feature be removed from the readme at all for this reason?

Reply:
I'm trying to connect to Kube here. https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/#quality-of-service-classes